modules/fscloud/variables.tf

```diff
@@ -203,6 +203,11 @@ variable "operating_system" {
   type        = string
   description = "The operating system of the workers in the default worker pool. If no value is specified, the current default version OS will be used. See https://cloud.ibm.com/docs/openshift?topic=openshift-openshift_versions#openshift_versions_available ."
   default     = null
+  validation {
```

Comment:
The fscloud submodule also has a `worker_pools` input map that would need the new operating_system property, just like main module

Reply:
sure, thanks Todd. Now added `operating_system` in `fscloud` module.